Lib/glyphsLib/classes.py

```diff
@@ -2787,13 +2787,15 @@ def _serialize_to_plist(self, writer):
     _parent = None
     _defaultsForName = {"position": Point(0, 0)}
 
-    def __init__(self, name=None, position=None):
+    def __init__(self, name=None, position=None, userData=None):
         self.name = "" if name is None else name
         self._userData = None
```

Comment:
we are setting `self.userData` immediately below. is this `self._userData` still needed?

Reply:
oh I see. then maybe move them closer together and add a small comment?

Reply:
It is a pattern used across the file, so I’m not sure a comment here only is neccassary.